shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/ct/SyTableReplication.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf.ct;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+import com.hotels.bdp.circustrain.api.conf.SourceTable;
+
+public class SyTableReplication {
+  private @Valid @NotNull SourceTable sourceTable;
```

Comment:
I think Source table should also be copied. less confusing

Reply:
Yeah. Thought about that but decided against. Now that you are saying, I have added that as well. Its named as `SySourceTable`.